### DISCUSSION TITLE
Headless Server Instance?

### DISCUSSION THREAD

**Discussion by chnl** [elizaOS/eliza#802] [Ideas]
Wanted to run this on headless Ubuntu Server. Ran into Chromium dependency, Is there an easy way to disable browser dependencies?

**Comment by internet-dot** [comment]
A headless mode would be useful for running Eliza agents as discoverable services. Combined with a public agent registry, headless agents could advertise their capabilities and be found by agents from other frameworks.

For example, using the [HOL Registry Broker](https://hol.org/registry) via their MCP server (`@hol-org/hashnet-mcp`), a headless Eliza agent could:
- Register itself with a UAID (stable identifier)
- Be discovered by LangChain, CrewAI, or A2A agents
- Receive messages through the registry P2P layer

This would make headless Eliza instances interoperable with the broader agent ecosystem without needing their own discovery protocol.
